### PR TITLE
LG-2553: Refresh confirmation modal

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-2553](https://jira.mongodb.org/browse/LG-2553)

Screen Shot 2022-11-07 at 4 55 30 PM
Screen Shot 2022-11-07 at 4 56 06 PM
Screen Shot 2022-11-09 at 4 00 42 PM
Screen Shot 2022-11-09 at 4 01 01 PM

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: packages/confirmation-modal/src/ConfirmationModal.tsx [hunk 1]
```diff
@@ -240,16 +93,12 @@ export const ConfirmationModal = ({
           variant={variant}
           disabled={!confirmEnabled || submitDisabled}
           onClick={onConfirm}
-          className={buttonStyle[mode]}
+          className={buttonStyle}
           darkMode={darkMode}
```
Comment (spark33) [inline packages/confirmation-modal/src/ConfirmationModal.tsx:97]: Does `darkMode` have to be redefined here?

File: packages/confirmation-modal/src/ConfirmationModal.tsx [hunk 2]
```diff
@@ -240,16 +93,12 @@ export const ConfirmationModal = ({
...
         >
           {buttonText}
         </Button>
-        <Button
-          onClick={onCancel}
-          className={buttonStyle[mode]}
-          darkMode={darkMode}
-        >
+        <Button onClick={onCancel} className={buttonStyle} darkMode={darkMode}>
```
Comment (spark33) [inline packages/confirmation-modal/src/ConfirmationModal.tsx:101]: Same thing here

**Review by spark33 — CHANGES_REQUESTED** [review]

File: packages/confirmation-modal/src/ConfirmationModal.tsx [hunk 3]
```diff
@@ -1,154 +1,25 @@
...
-  requiredInputText?: string;
-  /**
-   * If `true`, the primary action button will be disabled
-   */
-  submitDisabled?: boolean;
-  darkMode?: boolean;
-}
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { H3 } from '@leafygreen-ui/typography';
+import {
```
Comment (brookescarlett) [inline packages/confirmation-modal/src/ConfirmationModal.tsx:11]: ditto on the import * from

Comment (shaneeza) [reply packages/confirmation-modal/src/ConfirmationModal.tsx:11]: Gonna merge this in but left a comment about this in the Modal PR

**Pull request merged**